color/p5js/waveformcut/index.js

```diff
@@ -0,0 +1,65 @@
+var ball;
+var population;
+var from;
+var to;
+var esize = 2;
+var multiplkator = 4;
+
+function setup () {
+  //gradient from and to color
+  from = color(204, 102, 0);
+  to = color(0, 102, 153);
+  createCanvas(400,400);
```

Comment:
Canvas should be 500  × 500 and needs the assignment to the sketch div see https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code